### ISSUE TITLE
Output resolution does not match set resolution

### ISSUE THREAD

**Issue by MartLeib** [legraphista/node-server-screenshot#23]
OS: Windows
Node: v12.10.0

Running code:
```
var app = require("node-server-screenshot");
app.fromURL("https://google.com", "test.png", {
	width: 1600,
	height: 900,
	show: false,
}, function(){
    console.log("Saved");
});
```

Produces test.png with dimensions 1584x841

1920x1080 produces 1904x1021
2560x1440 produces 2544x1381
1280x720 produces 1264x661

Width loses 16px and height loses 59px. I would suspect that the dimensions set in the options do not change viewport size but set the size for the window that is captured?

**Comment by legraphista** [comment]
Hi @MartLeib 
Nice catch!

This module uses [nightmare](https://github.com/segmentio/nightmare), which in turn uses [electron](https://github.com/electron/electron).

While reading the electron doc [here](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#new-browserwindowoptions) it states that the width/height sets the size of browser window, which is why there is a discrepancy between requested size and output size.

Upon further reading i've found an option to enable [frameless windows](https://github.com/electron/electron/blob/master/docs/api/frameless-window.md) which (thankfully) nightmare passes along to electron. 
I've managed to reproduce and fix the issue, and will be pushing a update soon.

**Comment by legraphista** [comment]
Fix published in v0.2.4.
Feel free to re-open/comment this issue if necessary.
